README.md

```diff
@@ -1,27 +1,508 @@
 # React Changing State
 
-## Objectives
+## Overview
 
-1. Practicing updating state in React
-2. Practice passing state through props
-3. Describe how React rerenders on state changes
+React allows you to structure your application using small, reusable components,
+but React isn't really just about creating static pages, it's about building
+**react**ive user interfaces that enable you to coordinate complex user
+interactions.
 
-## Overview
+In this lesson we will be learning about how you can update the state of
+components. You are going to learn about the difference between `state` and
```

Comment:
At this point the students should already have learned about the difference between state and props